hnn_core/tests/test_io.py

```diff
@@ -0,0 +1,134 @@
+import os.path as op
+from numpy.testing import assert_allclose
```

Comment:
```suggestion
import os.path as op

from numpy.testing import assert_allclose
```

separate system builtin imports from other libraries